src/components/shared/inputOption.ts

```diff
@@ -0,0 +1,9 @@
+import { PlainTextElement } from '@slack/types';
+import { AnyText } from '../AnyText';
+
+export interface InputOption {
+  text: ReturnType<AnyText>;
+  value?: string;
+  description?: PlainTextElement;
```

Comment:
can this just be a string and you can build the plaintext object internally?

Reply:
sure, only thing we'll be missing is the option to set `emojis: true/false`, that should be fine

Reply:
yeah we can always abstract that if we need to tweak it. as long as we have a default that enables them